README.md

```diff
@@ -178,14 +200,47 @@ const useStoreB = create<StoreState>(
 );
 ```
 
+If converting temporal store to a React Store Hook with typescript, be sure to define the type of
```

Comment:
This is new -- giving an example of `useTemporalStore` with partialized state. If users use`TemporalState<StoreState>` while using partialize as a ZundoOption, they will see a type error. So this is to preempt that kind of error by providing an example of useTemporalState with a partalized store state typescript generic

Reply:
Wonderful!